shell/completion.bash

```diff
@@ -304,6 +304,13 @@ _fzf_alias_completion() {
 
 # fzf options
 complete -o default -F _fzf_opts_completion fzf
+if command -v fzf-tmux >/dev/null; then
+  complete -o default -F _fzf_opts_completion fzf-tmux
+  # fzf-tmux is a thin fzf wrapper that has only a few more options than fzf
+  # istself. As a quick improvement we'll take fzf's completion and adding the
+  # few extra fzf-tmux specific options (like `-w WIDTH`) leave as a future
+  # patch.
+fi
```

Comment:
I think it's okay to remove the if condtion.

```suggestion
# fzf-tmux is a thin fzf wrapper that has only a few more options than fzf
# itself. As a quick improvement we'll take fzf's completion.
# TODO: Add fzf-tmux specific options (e.g. `-w WIDTH`)
complete -o default -F _fzf_opts_completion fzf-tmux
```

Anyway, please note that the fzf completion hasn't been updated for quite a while, so it's missing a lot of new options.

Reply:
> I think it's okay to remove the if condition.

Done.

> Anyway, please note that the fzf completion hasn't been updated for quite a while, so it's missing a lot of new options.

Good to know. I'll leave that to some separate, future PR. Perhaps someone will have the energy to automate generation of the options (say for [lists like this](https://github.com/junegunn/fzf/blob/51fdaad002a5ad827bd5ebfac43386592005d02c/shell/completion.bash#L71-111) to not be hardcoded).